index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Mobile Portfolio</title>
+  <header>
+      <img src="img/My Logo.svg" alt="humburger" class="Logo">
+      <img src="img/Union.svg" alt="humburger"  class="Union" >
+  </header>
+</head><br><br><br><br>
+<body> 
+    <div class="container">
+      <h1>I'm Edgah.<br>
+        Glad to see<br>
+        you!</h1>
+      <p> I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+    <img src="img/LET’S CONNECT.svg" alt="humburger"><br><br>
+    </div>
+  <div>
+    <ul class="social-icons">
+      <li>
+          <img src="img/twitter icon.svg">
+      </li>
+      <li>
+          <img src="img/Linkedin icon.svg">
+      </li>
+      <li>
+          <img src="img/Mobile icon.svg">
+      </li>
+      <li>
+        <img src="img/git icon.svg">
+      </li>
+      <li>
+        <img src="img/peace icon.svg">
+      </li>
```

Comment:
- [ ] [OPTIONAL] These supposed links would have improve the accessibility of your page if if users can click on each of them and it leads to your social windows but they are not links at the moment since they are missing the `<a href="link_here">Name</a>`

For example:
```
<ul class="social-icons">
      <li>
           <a href="https://profile.com">
              <img src="img/peace icon.svg">
           </a>
      </li>
</ul>